templates/gandi/0/rancher-compose.yml

```diff
@@ -0,0 +1,37 @@
+.catalog:
+  name: "Gandi DNS"
+  version: "v0.2.1-rancher1"
```

Comment:
I thought this was the catalog service version, not the container version (which is 0.3.0)